packages/core/src/components/dialog/_dialog-footer.scss

```diff
@@ -0,0 +1,40 @@
+.#{$ns}-dialog-footer {
+  flex: 0 0 auto;
```

Comment:
Removed side margin. Will break things. I can add a class to keep it and then suppress it if using the new <DialogBody /> component.

Reply:
Yeah, we'll need to keep the existing margin and add any new functionality with a modifier class.